openselery/openselery.py

```diff
@@ -0,0 +1,291 @@
+import subprocess
+import argparse
+import os
+import json
+import yaml
+import random
+import sys
+
+from openselery.github_connector import GithubConnector
+from openselery.librariesio_connector import LibrariesIOConnector
+from openselery.coinbase_connector import CoinbaseConnector
+from openselery import selery_utils
+
+
+class OpenSeleryConfig(object):
+    __default_env_template__ = {
+        "libraries_api_key": 'LIBRARIES_API_KEY',
+        "github_token": 'GITHUB_TOKEN',
+        "coinbase_token": 'COINBASE_TOKEN',
+        "coinbase_secret": 'COINBASE_SECRET',
+    }
+    __default_config_template__ = {
+        "bitcoin_wallet": "",
+
+        "dryrun": True,
+        "include_dependencies": False,
+        "include_self": True,
+        "include_tooling_and_runtime": False,
+        "min_contributions": 1,
+        "check_equal_privat_and_public_wallet": True,
+        "skip_email": True,
+        "email_note": "Fresh OpenCelery Donation",
+        "btc_per_transaction": 0.000002,
+        "contributor_payout_count": 1,
+        "total_payout_per_run": 0.000002,
+    }
+    __secure_config_entries__ = ["libraries_api_key", "github_token", "coinbase_token", "coinbase_secret",
+                                 "coinbase_secret"]
```

Comment:
Why do these members start with `__` and end on it? I personally prefer to avoid `_` both at the start and at the end of any `identifier` if possible.

Reply:
yeah i dont like it either, because python has no real way of hiding private members i usually "hide" them by putting them into a context which is not usually a concern for people using the api, except for when they REALLY want to break stuff.

in this case, members starting with _ or __ are not usefull for external use (outside of known context) 

messing aroudn with this specific dict is not really usefull nor helpful, this is essentially not really necesary nor consistent but it helps seperating user APIs for internal stuff (in other known libraries aswell)